build-image.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
+if [ $1 == "fast" ]; then
```

Comment:
Suggestion: Instead, of "fast", perhaps "nocompress"? ie something a little more precise and clearer.

Reply:
633badf308b8a326225ae3cb4f2a281ab93f57c8